index.bs

```diff
@@ -3356,6 +3357,9 @@ Note: The {{AuthenticatorTransport}} enumeration is deliberately not referenced,
     :   <dfn>ble</dfn>
     ::  Indicates the respective [=authenticator=] can be contacted over Bluetooth Smart (Bluetooth Low Energy / BLE).
 
+    :   <dfn>hybrid</dfn>
+    ::  Indicates the respective [=authenticator=] can be contacted over the Hybrid transport that combines (often separate) data-transport and proximity mechanisms. This is most commonly used to communicate with phones.
```

Comment:
> ...can be contacted over the Hybrid transport that combines (often separate) data-transport and proximity mechanisms. This is most commonly used to communicate with phones.

How about something like this instead?

> ...can be contacted using a combination of (often separate) data-transport and proximity mechanisms. This supports, for example, authentication on a desktop using a previously registered smartphone.

I think it's important to leave examples here open-ended because we've seen Apple demonstrate caBLE QR code display on an iPhone, ostensibly for cross-device auth (I haven't tested it myself but that was the impression the WWDC session left on me)